ale_linters/chef/foodcritic.vim

```diff
@@ -30,10 +31,15 @@ function! ale_linters#chef#foodcritic#Handle(buffer, lines) abort
     return l:output
 endfunction
 
+" Support options!
+let g:ale_chef_foodcritic_options = get(g:, 'ale_chef_foodcritic_options', '')
+
 call ale#linter#Define('chef', {
 \   'name': 'foodcritic',
 \   'executable': 'foodcritic',
-\   'command': 'foodcritic %t',
+\   'command': 'foodcritic '
+\       . g:ale_chef_foodcritic_options
+\       . ' %t',
```

Comment:
It might be an idea to add a test case for this, too, once the callback is in. There are a few existing tests like that. Maybe it's a bit excessive, but it can't hurt.